nautobot_golden_config/templates/nautobot_golden_config/content_template.html

```diff
@@ -121,4 +124,3 @@
   </div>
   {% endif %}
 </div>
-{% endif %}
```

Comment:
This seems off, shouldn't have a div end everything if it starts with a conditional, which should capture everything.

Reply:
this is fixed now and tested.